src/box/lua/schema.lua

```diff
@@ -1292,142 +1416,33 @@ function box.schema.space.bless(space)
         return box.schema.index.alter(index.space_id, index.id, options)
     end
 
+    index_mt.__pairs = index_mt.pairs -- Lua 5.2 compatibility
+    index_mt.__ipairs = index_mt.pairs -- Lua 5.2 compatibility
+
+    return index_mt
+end)()
+
+function box.schema.space.bless(space)
+    local index_mt = {}
+    index_mt.__index = function(index, key)
+      return index_mt[key] or box.index_mt[key]
```

Comment:
Why do you need index_mt[key], if all of your extensions must be in box.schema.index_mt ?